internal/cloud/api.go

```diff
@@ -176,6 +176,50 @@ func (a *api) DeleteSSL(clusterID, sslID cloud.ID) error {
 	})
 }
 
+func (a *api) ListSSL(clusterID cloud.ID, limit int, skip int) ([]*cloud.CertificateDetails, error) {
```

Comment:
Maybe we should mention that the value of `limit` should be less than 25 due to how you have implemented this List function?